src/Orchard/Mvc/Html/LayoutExtensions.cs

```diff
@@ -48,6 +48,16 @@ public static class LayoutExtensions {
             return MvcHtmlString.Create(html.Encode(titleParts[0]));
         }
 
+        public static MvcHtmlString TitleForPage(this HtmlHelper html, params MvcHtmlString[] titleParts) {
+            if (titleParts == null || titleParts.Length < 1)
+                return null;
+
+            string[] parts = titleParts.Select(t => t.ToString()).ToArray();
+            html.AppendTitleParts(parts);
+
+            return MvcHtmlString.Create(html.Encode(titleParts[0]));
```

Comment:
An HtmlString is supposed to be encoded already, it should not be encoded again.